uefi-sct/SctPkg/Include/SctLib.h

```diff
@@ -132,6 +132,16 @@ SctDevicePathStrFromProtocol (
   IN EFI_GUID    *Guid
   );
 
+struct _EFI_STANDARD_TEST_LIBRARY_PROTOCOL;

```

Comment:
There should be no need to duplicate these two lines for the EFI_STANDARD_TEST_LIBRARY_PROTOCOL. 
Could you add gEfiStandardTestLibraryGuid to SctLib.inf instead and see if it works? If not, can we directly include "Protocol/StandardTestLibrary.h"?

Reply:
Hi Sunny
Adding the StandardTestLibrary header file is causing redeclaration issues, hence I included it in devicepath.c and made a forward declaration in sctlib.h

Reply:
In this case, if we want to use the wrapper function, we need to figure out what is wrong with the StandardTestLibrary header file. 
Again, I'm fine with NOT using the wrapper function now as it is more complicated than I thought.

Reply:
The issue, this was declared in multiple header files, tracing dependency was a bit challenging hence had to go with this approach:
typedef enum {
  EFI_TEST_ASSERTION_PASSED,
  EFI_TEST_ASSERTION_WARNING,
  EFI_TEST_ASSERTION_FAILED
} EFI_TEST_ASSERTION;
If we do not want to use a wrapper function we can roll back to the previous approach where we were only calling SctDevicePathStrFromProtocol and use recordmessage within the test function.

Reply:
Agree, so let's roll back to the previous approach.